source/deimos/openssl/kdf.di

```diff
@@ -142,51 +142,51 @@ else
 	auto EVP_PKEY_CTX_set_tls1_prf_md () (EVP_PKEY_CTX* ctx, const(EVP_MD)* md)
 	{
 		return EVP_PKEY_CTX_ctrl(
-			pctx, -1, EVP_PKEY_OP_DERIVE, EVP_PKEY_CTRL_TLS_MD, 0, md);
+			ctx, -1, EVP_PKEY_OP_DERIVE, EVP_PKEY_CTRL_TLS_MD, 0, md);
 	}
 
 	auto EVP_PKEY_CTX_set1_tls1_prf_secret () (
-		EVP_PKEY_CTX* pctx, const(ubyte)* sec, int seclen)
+		EVP_PKEY_CTX* ctx, const(ubyte)* sec, int seclen)
```

Comment:
Not that it affects anything but the argument name is `pctx` in the C headers, we should stick close to them whenever possible.